src/components/Meme.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useState,useEffect } from "react";
+export default function Meme (){
+
+  const [memeText, setMemeText] = useState({
+    firstText: "",
+    secondText: "",
+    randomImage: "http://i.imgflip.com/1bij.jpg",
+  });
+
+  function handleChange(event){
+    const {name,value}= event.target
+    setMemeText(prev => ({...prev, [name]:value}))
+  }
+
+  const[allMeme, setAllMeme]=useState([])
```

Comment:
Moving this state to the top of the function will improve the formatting of your code.